src/main/client/java.client.ftl

```diff
@@ -303,6 +310,17 @@ public class FusionAuthClient {
     return new FusionAuthClient(apiKey, baseURL, connectTimeout, readTimeout, tenantId.toString());
   }
 
+  /**
+  * Creates a new copy of this client with the object mapper. This will take the place of the default FusionAuth object mapper when serializing
+  * objects to JSON for the request body
```

Comment:
Technically we will use this for serialization on the API request and deserialization for the API response.